### ISSUE TITLE
Shortcuts don't honor modifier keys

### ISSUE THREAD

**Issue by dragonpaw** [Inventory-Tweaks/inventory-tweaks#600]
**Minecraft Version: 1.12.2**

**Inventory Tweaks Version: 1.64-dev**

When using the in-game controls to change the sort key, modifiers seem to be ignored. If I set the control to CTRL-r for example, pressing r alone still sorts my inventory. (As does CTRL-r.) But setting it to an unmodified key like ScrollLock works fine.

**Comment by Spongman** [comment]
yeah, please fix this. the 'R' keybinding triggers in refined storage causing the hotbar to be sorted when searching for anything containing the letter 'R'. i don't know whose bug this is, but rebinding to a key combo using a modifier (eg CTRL) is the obvious workaround. except it doesn't work.
